scripts/update_comments.py

```diff
@@ -0,0 +1,41 @@
+"""
+Update User.comments_viewed_timestamp field & comments model.
+Accompanies https://github.com/CenterForOpenScience/osf.io/pull/1762
+"""
+from modularodm import Q
+from framework.auth.core import User
+from website.models import Comment
+from website.app import init_app
+import logging
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    init_app(routes=False, set_backends=True, mfr=False)
```

Comment:
The `mfr` parameter has been removed from `init_app`, so it needs to be removed here.